src/components/Modal/Translate.vue

```diff
@@ -167,6 +153,11 @@ export default {
 			this.error = null
 			this.autosize()
 		},
+		fromLanguage(newVal) {
+			if (newVal.id === this.toLanguage.id) {
+				this.toLanguage = null
+			}
+		},
```

Comment:
* Fix the errors thrown in line 157 by handling `null` as `newVal` and `this.toLanguage` well.
* When changing `fromLanguage` - also clear the result so it can be recomputed.
```suggestion
		fromLanguage(newVal) {
			if (newVal?.id === this.toLanguage?.id) {
				this.toLanguage = null
			}
			this.result = ''
			this.error = null
		},
```